repo/fsrepo/fsrepo.go

```diff
@@ -578,6 +581,26 @@ func (r *FSRepo) GetStorageUsage() (uint64, error) {
 	return du, err
 }
 
+func (r *FSRepo) SwarmKey() ([]byte, error) {
```

Comment:
Config is exposed via the HTTP API and right now we some ugly hacks to not expose the privkey over the API.